site/app/templates/officeHoursQueue/QueueFooter.twig

```diff
@@ -33,11 +33,12 @@
         $("#queue_filter_" + current_queues[i]).prop("checked", true);
         filter_choices[current_queues[i]] = true;
       }
-      if (filter_choices[current_queues[i]]) {
-          document.getElementById('queue_filter_' + current_queues[i]).parentElement.classList.remove("fully-transparent");
+      var queue_filter = document.getElementById('queue_filter_' + current_queues[i]);
+      if (filter_choices[current_queues[i]] && queue_filter != null) {
```

Comment:
With #6350, we should not need to check on `queue_filter != null` here or in the else if anymore.

Additionally, I would recommend writing the these lines (if we were to keep the null check) like:
```
      if (queue_filter == null) {
        continue;
      }
      if (filter_choices[current_queues[i]]) {
          queue_filter.parentElement.classList.remove("fully-transparent");
      }
      else {
          queue_filter.parentElement.classList.add("fully-transparent");
      }
```
So that we only have the conditional check on `queue_filter` in one place instead of two, or just wrapping the if/else inside an if check of `if (queue_filter != null)`

Reply:
I took a look at #6350 and it appears to disable my timer. Simply put, I noticed that filterQueues() was being called on the QueueStatus page as well so I simply re-used the already present timer code for the student's side. I'll probably just need to separate some of the timer code from filterQueues() so it can work as before.